### DISCUSSION TITLE
sea-orm Mock feature current now not ergonomics

### DISCUSSION THREAD

**Discussion by baoyachi** [SeaQL/sea-orm#1201] [General]
As so far, we tried to use it, but it didn't meet expectations

# why
* mock not support clone. 
* write Test unit not inconvenient when call db function .

**Comment by billy1624** [comment]
Hey @baoyachi, I understand the `Clone` issue. But could you elaborate more on the API of mock testing? Why it's inconvenient? Can you provide a code snippet / demo repository? Thanks!!
